### DISCUSSION TITLE
Manipulate data from Users built-in table

### DISCUSSION THREAD

**Discussion by laromana** [Budibase/budibase#6461] [Help]
I am not sure if I am missing something here but I have been trying for days and I can't find a way to figure this one out.

I am trying to pull some data from the Users built-in table, I am trying to grab the user's First name and Last name to create a column to then allow users to pick their names from the list of options, so far I am able to create a relationship column and and relationship picker, so far so good, I get all the users listed but I also get the Admin users (some of them even blanks options), so how can I narrow the list of options to only the ones I need? I read that it is not possible to filter a relationship column so how can I accomplish this then?

Thank you for your help in advance, I have been trying to days and I just feel like giving up at this point.

**Comment by melohagan** [comment]
Hi @laromana 

I'm not fully understanding what you are trying to achieve. Would you mind providing some screenshots please?

**Reply by laromana** [reply]
Sorry I didn't explain myself very well.

I have the two tables below, I want to display the FULL_NAME column from the Users table as a list of choices in a form, I am trying to use the relationship picker component with the column MANAGER as the FIELD, I want to be able to filter the list of options but I am getting all Users instead, I want users to be able to pick only users listed as MANAGERS from the Users built-in table. 

The relationship is:

Table Users:
     Display column = FULL_NAME
     Relationship column = Demo_user
Table Demo:
    Display column = NAME
    Relationship column = MANAGER

![Untitled presentation (6)](https://user-images.githubusercontent.com/79430611/176021263-1ece035f-35b6-4014-b5d3-58dc4cc291b0.png)

![Untitled presentation (5)](https://user-images.githubusercontent.com/79430611/176021678-ad234a30-a7a4-4a6b-a302-031c185a9295.png)

Thank for your replay!! Hope this makes it a bit clearer

**Comment by melohagan** [comment]
Hi @laromana 

Thank you very much for the additional information - I understand the problem now.

This appears to be a limitation with the Relationship picker. Unlike the Options Picker which allows you to specify an *Options source*, with the Relationship Picker it automatically assigns a source from the source table of the relationship, which in this case is all of the users.
There is no way to directly filter this.
